tests/lit-tests/1319.ispc

```diff
@@ -1,4 +1,4 @@
-//; RUN: %{ispc} %s -h %t.h -o /dev/null
+//; RUN: %{ispc} %s -h %t.h --disable-vectorcall -o /dev/null
```

Comment:
``-o /dev/null`` is not needed here and can be removed.
I suggest instead of adding ``--disable-vectorcall`` modifying a check to allow ``__vectorcall`` in the string.